libraries/rush-lib/src/logic/Telemetry.ts

```diff
@@ -41,6 +55,9 @@ export interface ITelemetryData {
    */
   readonly rushVersion?: string;
   readonly extraData?: { [key: string]: string | number | boolean };
+  readonly machineInfo?: IMachineInfo;
+  readonly buildTimings?: IBuildTimeRecord[];
+  readonly dependencyGraph?: IDependencyGraph;
```

Comment:
Why are these separate objects? We could just list the timings and status values on the graph nodes.